README.md

```diff
@@ -114,3 +114,18 @@ Any kind of contribution to ChainerRL would be highly appreciated! If you are in
 ## License
 
 [MIT License](LICENSE).
+
+## Citations
+
+If you use ChainerRL for your research, we ask that you cite the following paper:
+
+```
+@InProceedings{fujita2019chainerrl,
+  author = {Fujita, Yasuhiro and Kataoka, Toshiki and Nagarajan, Prabhat and Ishikawa, Takahiro},
+  title = {ChainerRL: A Deep Reinforcement Learning Library},
+  booktitle = {NeurIPS 2019 Deep Reinforcement Learning Workshop},
```

Comment:
It's possible that `booktitle` starts with "Proceedings of" and contains "33rd", etc.  Could you check the formal title of the proceedings?

Reply:
There is no formal title for workshop proceedings.

Reply:
Or are you referring to the formal NeurIPS proceedings?

Reply:
Thanks.  I got it.